src/main/java/com/vinyl/model/Token.java

```diff
@@ -0,0 +1,55 @@
+package com.vinyl.model;
+
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.validation.constraints.NotNull;
+
+import org.hibernate.annotations.GenericGenerator;
+
+
+@Entity
+public class Token {
```

Comment:
a liquibase script that creates the table?

Reply:
done